h/views/admin_groups.py

```diff
@@ -45,18 +45,36 @@ def get(self):
     @view_config(request_method='POST')
     def post(self):
         def on_success(appstruct):
-            read_url = self.request.route_url('admin_groups')
-            self.request.session.flash('TODO: I will add a {gtype} group called "{name}"'
-                                       ' for authority {authority}, created by {creator}'
-                                       ' and origins {origins}'.format(
-                                            gtype=appstruct['group_type'],
-                                            name=appstruct['name'],
-                                            authority=appstruct['authority'],
-                                            creator=appstruct['creator'],
-                                            origins=', '.join(appstruct['origins'])
-                                       ), queue='success')
-            response = HTTPFound(location=read_url)
-            return response
+            svc = self.request.find_service(name='group')
+
+            # Create the new group.
+            creator = appstruct['creator']
+            authority = appstruct['authority']
+            description = appstruct['description']
+            name = appstruct['name']
+            origins = appstruct['origins']
+            type_ = appstruct['group_type']
+
+            userid = 'acct:{}@{}'.format(creator, authority)
```

Comment:
> If I enter acct:seanh@hypothes.is instead of just seanh into the user field I get a confusing error message:

Given that the userid format is a mostly internal/API-level thing and that we use ordinary usernames everywhere else in the admin panel, I think it quite unlikely that a user is going to try and do this.

I do think it would be helpful to accept either a username or email address here though, but I think we can address that separately.

Reply:
Ok, happy to proceed with only `seanh` working for now